operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -648,11 +648,8 @@ function generateMergedModel() {
     ${archive_list} ${variable_list}  -domain_type ${WDT_DOMAIN_TYPE}  > ${WDT_OUTPUT}
   ret=$?
   if [ $ret -ne 0 ]; then
-    trace SEVERE "WDT Failed: Validate Model Failed "
-    if [ -d ${LOG_HOME} ] && [ ! -z ${LOG_HOME} ] ; then
-      cp  ${WDT_OUTPUT} ${LOG_HOME}/introspectJob_validateDomain.log
-    fi
-    trace SEVERE "$(cat ${WDT_OUTPUT})"
+    trace SEVERE "WDT Failed: Validate Model Failed:"
+    cat ${WDT_OUTPUT}
```

Comment:
Implementation note: we no longer need to check if LOG_HOME is set and/or do the 'cp' because the calling script is now logging the entire output of this script.